lib/public/components/Post/index.js

```diff
@@ -12,19 +12,63 @@
  */
 import { h } from '/js/src/index.js';
 
+const ACTIVE_FIELDS = {
+    id: {
+        name: 'ID',
+        visible: true,
+    },
+    authorId: {
+        name: 'Source',
+        visible: true,
+    },
+    creationTime: {
+        name: 'Created',
```

Comment:
Please verify that the datatype test (frontend tests) in the public folder still works

Reply:
The test for the datatype tests still works (as can be seen by GH actions), if that is what you meant.